src/Esfa.Vacancy.Register.Domain/Entities/GeoPoint.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Esfa.Vacancy.Register.Domain.Entities
+{
+    public class GeoPoint
+    {
+        public double? Lon { get; set; }
```

Comment:
I think it is common to abbreviate Longitude to Long. Normally people say 'Lat/Long'. I don't know if you shortened it because it may conflict with the Long C# type/keyword.

Reply:
This is how it is stored in the ES, hence just matched it up to deserialise without any faffing around. The object that is exposed via API has full names.